spec/isPalindrome.js

```diff
@@ -0,0 +1,19 @@
+import { expect } from 'chai'
+import isPalindrome from '../src/isPalindrome'
+
+
+describe('isPalindrome()', () => {
+'use strict'
+
+  it('exists', () => {
+    expect(isPalindrome).to.be.a('function')
+  })
+  it('should return true if string is a palindrome', () => {
+    expect(isPalindrome('tacocat')).to.equal(true)
+    expect(isPalindrome('radar')).to.equal(true)
+    expect(isPalindrome('bananas')).to.equal(false)
```

Comment:
Part of the spec: Ignore punctuation, spacing, and **case sensitivity**.

Try testing for `rAdar`.